pkg/client/client_venconn.go

```diff
@@ -86,10 +86,13 @@ func NewVenConnClient(restcfg *rest.Config, agentMetadata *api.AgentMetadata, in
 	_ = venapi.AddToScheme(scheme)
 	_ = corev1.AddToScheme(scheme)
 
+	var unusedTPPDefaultClientID string
+
 	handler, err := venafi_client.NewConnectionHandler(
 		version.UserAgent(),
 		"venafi-kubernetes-agent.jetstack.io",
 		"VenafiKubernetesAgent",
+		unusedTPPDefaultClientID,
```

Comment:
This new argument was introduced in: https://github.com/jetstack/venafi-connection-lib/pull/316
* https://github.com/jetstack/venafi-connection-lib/pull/316